### DISCUSSION TITLE
Feature Request: Empty README files should not report an error

### DISCUSSION THREAD

**Discussion by oliveryasuna** [community/community#113934] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If a repository has an empty README...

![image](https://github.com/community/community/assets/17092333/91533adb-89c6-4589-ae31-945741dac6de)

I think it should say something like "The readme file is empty."

**Comment by IsmaelMousa** [comment]
No it's okay, when a repository has an empty `README.md` you will got the same message so no worries.

**Comment by arsalanmughal23** [comment]
when you upload your project you can maintain your readme.md file like this given below screenshot
by just place this content in README.md file and modifiy project name and details according to your project
```# Generative AI Cource @TypeScript
  > #typescript | First 45 Assignments

> [!NOTE]
> ## How to Run Task Project
> Go to the task directory inside the terminal, **example** is listed below
  > ```
  >  cd ./task_2
  > ```
> **Then!** Run the project by using given below command
  > ```
  > npm run start
  > ```

> [!TIP]
> ## How to go back step directory
  > ```
  > cd ../
  > ```
 
### 45 Tasks <sup>(45 / 45)</sup>
1. Install Node.js, TypeScript and VS Code on your computer. ✔
2. Personal Message: Store a person’s name in a variable, and print a message to that person. Your message should be simple, such as, “Hello Eric, would you like to learn some Python today?” ✔
```

And you should also view my readme.md file from this link: 
**Repository Link:** https://github.com/arsalanmughal23/genAi_ts_first45Assignments

if you have any query feel free to ask.

![image](https://github.com/community/community/assets/50748011/99f03cf2-82ee-4560-ae71-73b793eac732)
